test/discretization.jl

```diff
@@ -1,4 +1,13 @@
 @testset "Discretization" begin
+  function is_fully_connected(mesh::UnstructuredMesh)
+    inds = collect(1:length(vertices(mesh)))
+    connected = collect(Iterators.flatten(getproperty.(mesh.connec, :list)))
```

Comment:
We need to refactor this line because it is using internal behavior. Why not just compare the `Connectivity` objects directly? We shouldn't need to access the internal list of integers.

Reply:
Also it seems that the variable `connected` is not used later in the function. A typo?

Reply:
Yes, that was a big typo, we have to compare to `connected`. We need the indices of each `Connectivity` object, to make sure they include all indices of the mesh. AFAIK there is no public way of doing that, which is why I rely on the `list` field. Since this is in tests, I don't think this poses problems, right? I'd rather avoid adding a public method in this PR that already eventually changes a lot of things.